pkg/envtest/envtest_test.go

```diff
@@ -65,15 +65,15 @@ var _ = Describe("Test", func() {
 				Name: crd.GetName(),
 			}
 			var placeholder v1beta1.CustomResourceDefinition
-			err := c.Get(context.TODO(), crdObjectKey, &placeholder)
+			err = c.Get(context.TODO(), crdObjectKey, &placeholder)
```

Comment:
```suggestion
			err := c.Get(context.TODO(), crdObjectKey, &placeholder)
```

Please do not do scope changes for variables like this (This comment doesn't only apply to this line but to all places where you did that).

Reply:
Got it. Will change it back. Did that as the part of go-lint with `no new variable`

Reply:
not sure what `no new variable` is but IMHO it is best to keep the scope for variables as small as possible and this kind of change is the very opposite to that. Reduces the odds of accidentally propagating values to the wrong place and makes it easier for the garbage collector